packages/app-degree-pages/src/core/services/degree-data-prop-resolver-service.js

```diff
@@ -138,6 +138,11 @@ function degreeDataPropResolverService(row = {}) {
     getEmailAddress: () => row["emailAddr"],
     /** @return {string} */
     getPhone: () => row["phoneNumber"]?.replace("/", "-"),
+    getProfessionalLicensureText: () => {
+      const standardText = row["professionalLicensureStandardText"] || "";
+      const additionalText = row["professionalLicensureAdditionalText"] || "";
+      return `${standardText}${additionalText}`;
+    },
```

Comment:
For historical record, we discussed and decided to go with the DOMParser approach for now. We prefer to have data issues solved at the data source, rather than have our  consumers implement heavy measures to address other's issues.